mathjax3-ts/input/tex/FindTeX.ts

```diff
@@ -32,208 +32,210 @@ import {MathItem, ProtoItem, protoItem, Location} from '../../core/MathItem.js';
 export type EndItem = [string, boolean, RegExp];
 export type Delims = [string, string];
 
-/*****************************************************************/
-/*
+/**
  *  Implements the FindTeX class (extends AbstractFindMath)
  *
  *  Locates TeX expressions within strings
  */
 
-/*
+/**
  * @template N  The HTMLElement node class
  * @template T  The Text node class
  * @template D  The Document class
  */
 export class FindTeX<N, T, D> extends AbstractFindMath<N, T, D> {
 
-    public static OPTIONS: OptionList = {
-        inlineMath: [              // The start/end delimiter pairs for in-line math
+  /**
+   * @override
+   */
+  public static OPTIONS: OptionList = {
+    inlineMath: [              // The start/end delimiter pairs for in-line math
```

Comment:
I had used 2-space indenting in v2, but the emacs mode for typescript defaulted to 4 (I think), so I've been using that her since.  All my v3 code is indented to 4 spaces.  I could do it either way, but had thought we were doing 4.

Reply:
You are absolutely right: v2 does use 2 space indentation. So I have no idea how the 4 space indentation comes from that ocassionally creeps into my code.
And I realise now that my indentation is globally set in my .emacs for all modes. So there is no setting in the js/ts modes. Really sorry about only realising this now. 
We can go back to 2 spaces, as it will preserve screen real estate.
Maybe for the next beta we can have one PR that does that?

Reply:
OK.  We are going to need to start keeping track of the global change PR's that we need.  I know we've mentioned several already.

Reply:
I'll start an issue tracker.